lxd/instances_post.go

```diff
@@ -1083,6 +1016,18 @@ func instancesPost(d *Daemon, r *http.Request) response.Response {
 				}
 			}
 
+			allMembers, err := tx.GetNodes(ctx)
```

Comment:
On the line above that says:

```go
if clustered && !clusterNotification && targetMemberInfo == nil {
```

I'm confused now about the ordering of `targetMemberInfo, err = project.CheckTarget` as it seems that its impossible for `targetMemberInfo` to be non-nil at this point.